data/json/start_locations.json

```diff
@@ -5,6 +5,96 @@
     "name": "Shelter",
     "target": "shelter"
   },
+  {
+      "type": "start_location",
+      "ident": "road_end_south"
+      "name": "Road End (South)"
```

Comment:
This whole file is missing commas after ident and name

Reply:
I think it's done. If there are any more, let me know.